vercel.json

```diff
@@ -1173,6 +1173,11 @@
       "source": "/docs/hydra/5min-tutorial",
       "destination": "/docs/hydra/self-hosted/quickstart",
       "permanent": false
+    },
+    {
+      "source": "/docs/quickstart/sdks",
```

Comment:
Please double check the vercel.json file before submitting PRs for review - it's missing the guides/authentication redirect. 404s in docs are very bad for SEO and we must avoid them at all costs. I know it's tedious but please review the vercel.json with deleted/changed URLs yourself before submitting for review, it's highly appreciated